test/client-request.js

```diff
@@ -228,6 +228,31 @@ test('request text', (t) => {
   })
 })
 
+test('request blob', (t) => {
```

Comment:
'should set type of blob object to the value of the `Content-Type` header from response'